### ISSUE TITLE
No error message displayed in webUI

### ISSUE THREAD

**Issue by adleyva** [getryft/ryft-apidemo#68]
the following error is not being handled by webUI.  

```
time="2017-01-27T11:48:06-05:00" level=info msg="[ryftdec]: decomposed as" input="bank of america" output="(RAW_TEXT CONTAINS EXACT(\"bank of america\", LINE=\"true\", CASE=\"false\"))[es,line,!cs]" task=dec-00000007
time="2017-01-27T11:48:06-05:00" level=info msg="[ryftprim]: start /search" cfg=Config{query:(RAW_TEXT CONTAINS EXACT("bank of america", LINE="true", CASE="false")), files:["regression/wikipedia-20150518.bin"], mode:"g", width:-1, dist:0, cs:false, nodes:0, limit:10000, keep-data:"", keep-index:"", delim:"", index:true, data:true} task=0000000000000007
time="2017-01-27T11:48:06-05:00" level=info msg="[ryftprim]: executing tool" args=[-p g -q (RAW_TEXT CONTAINS EXACT("bank of america", LINE="true", CASE="false")) -f regression/wikipedia-20150518.bin -e  -v -l -oi .rest-8765/.idx-0000000000000007.txt -od .rest-8765/.dat-0000000000000007.bin] task=0000000000000007 tool="/usr/bin/ryftprim"
time="2017-01-27T11:48:06-05:00" level=debug msg="[ryftprim]: start TASK..." task=0000000000000007
time="2017-01-27T11:48:06-05:00" level=debug msg="[ryftprim]: waiting for tool finished..." task=0000000000000007
time="2017-01-27T11:48:25-05:00" level=warning msg="[ryftprim]: tool failed" error="exit status 156" task=0000000000000007
time="2017-01-27T11:48:25-05:00" level=warning msg="[ryftprim]: tool output:\nERROR:  rcab1 received 1 flush error\n\n" task=0000000000000007
time="2017-01-27T11:48:25-05:00" level=debug msg="[ryftprim]: end TASK" result=Result{records:0, errors:1, done:true, cancelled:false, no stat} task=0000000000000007
time="2017-01-27T11:48:25-05:00" level=info msg="[core]: /search done" result=Result{records:0, errors:1, done:true, cancelled:false, no stat}
time="2017-01-27T11:48:25-05:00" level=warning msg="Panic recover: error" error="ryftprim failed with exit status 156\nERROR:  rcab1 received 1 flush error\n\n"
time="2017-01-27T11:48:25-05:00" level=debug msg="stack trace:\ngoroutine 5513 [running]:\nruntime/debug.Stack(0xc420fed4f0, 0x55b0293ce551, 0x14)\n\t/usr/local/go/src/runtime/debug/stack.go:24 +0x79\ngithub.com/getryft/ryft-server/rest.RecoverFromPanic(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/getryft/ryft-server/rest/error.go:99 +0x4bf\npanic(0x55b0292dd2c0, 0xc4206b6300)\n\t/usr/local/go/src/runtime/panic.go:458 +0x243\ngithub.com/getryft/ryft-server/rest.(*Server).DoSearch(0xc42012b600, 0xc42007ebb0)\n\t/home/dataart/go/src/github.com/getryft/ryft-server/rest/search.go:249 +0x1678\ngithub.com/getryft/ryft-server/rest.(*Server).DoSearch-fm(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/getryft/ryft-server/ryft-server.go:276 +0x34\ngithub.com/gin-gonic/gin.(*Context).Next(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/gin-gonic/gin/context.go:97 +0x5a\ngopkg.in/appleboy/gin-jwt%2ev1.(*GinJWTMiddleware).middlewareImpl(0xc4201df180, 0xc42007ebb0)\n\t/home/dataart/go/src/gopkg.in/appleboy/gin-jwt.v1/auth_jwt.go:135 +0x278\ngopkg.in/appleboy/gin-jwt%2ev1.(*GinJWTMiddleware).MiddlewareFunc.func1(0xc42007ebb0)\n\t/home/dataart/go/src/gopkg.in/appleboy/gin-jwt.v1/auth_jwt.go:113 +0x34\ngithub.com/getryft/ryft-server/middleware/auth.(*Middleware).Authentication.func1(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/getryft/ryft-server/middleware/auth/auth.go:134 +0x1a3\ngithub.com/gin-gonic/gin.(*Context).Next(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/gin-gonic/gin/context.go:97 +0x5a\ngithub.com/getryft/ryft-server/middleware/cors.Cors.func1(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/getryft/ryft-server/middleware/cors/cors.go:20 +0x27a\ngithub.com/gin-gonic/gin.(*Context).Next(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/gin-gonic/gin/context.go:97 +0x5a\ngithub.com/gin-gonic/gin.RecoveryWithWriter.func1(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/gin-gonic/gin/recovery.go:45 +0x5a\ngithub.com/gin-gonic/gin.(*Context).Next(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/gin-gonic/gin/context.go:97 +0x5a\nmain.main.func2(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/getryft/ryft-server/ryft-server.go:199 +0x91\ngithub.com/gin-gonic/gin.(*Context).Next(0xc42007ebb0)\n\t/home/dataart/go/src/github.com/gin-gonic/gin/context.go:97 +0x5a\ngithub.com/gin-gonic/gin.(*Engine).handleHTTPRequest(0xc4200ca9a0, 0xc42007ebb0)\n\t/home/dataart/go/src/github.com/gin-gonic/gin/gin.go:284 +0x64e\ngithub.com/gin-gonic/gin.(*Engine).ServeHTTP(0xc4200ca9a0, 0x55b0298fab20, 0xc4201b3a00, 0xc4201e9860)\n\t/home/dataart/go/src/github.com/gin-gonic/gin/gin.go:265 +0x130\nnet/http.serverHandler.ServeHTTP(0xc420139000, 0x55b0298fab20, 0xc4201b3a00, 0xc4201e9860)\n\t/usr/local/go/src/net/http/server.go:2202 +0x7d\nnet/http.(*conn).serve(0xc420081300, 0x55b0298fb920, 0xc420497d40)\n\t/usr/local/go/src/net/http/server.go:1579 +0x4b7\ncreated by net/http.(*Server).Serve\n\t/usr/local/go/src/net/http/server.go:2293 +0x44d\n"
```

![1-27-2017 11-56-18 am](https://cloud.githubusercontent.com/assets/14016898/22379657/27f6447a-e488-11e6-86bf-16543f37f7ef.png)

**Comment by dendec** [comment]
This issue was fixed in PR #69

**Comment by adleyva** [comment]
verified
